content/events/kick-off.yaml

```diff
@@ -13,11 +13,11 @@
 
     Our goal is for everybody to have a clear understanding about TechLabs Berlin. We will share the link to join here a few days before the event.
 
-  forms: 
-  - title: Feedback
-    description: Please leave feedback for our Kick-Off Event!
-    url: https://techlabsorg.typeform.com/to/bEsJr5Gh
-  resources: 
+  #  forms: 
```

Comment:
Interesting this actually breaks the radar. Did you run it locally?

Reply:
For fixing this use the following format instead of commenting it out. 
```
- forms:
    []
- resources:
    []
```

because

```
- forms:

- resources:

```
is not valid yaml & the radar expects both attributes :)

Reply:
Yes, also broke for me, I'm going to push it updated.